src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -322,6 +324,7 @@ export const AnnotationForm: FC<Props> = (props: Props) => {
                 invalidMessage={getEndTimeValidationMessage()}
                 onValidityCheck={setEndTimeFormatValid}
                 time={endTime}
+                timeFormat={props.timeFormat}
```

Comment:
this file has `timeFormat` as an optional param, but it doesn't look like the code ever handles it not existing. What happens if `timeFormat` is `undefined` ?

Reply:
Just fixed it :), I put a check in the AnnotationForm to see if it is defined, if not use our `DEFAULT_TIME_FORMAT` constant.